server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java

```diff
@@ -150,7 +151,7 @@ public static TSStatus tryCatchNonQueryException(Exception e) {
       // ignore logging sg not ready exception
       if (!(rootCause instanceof StorageGroupNotReadyException)) {
         if (((IoTDBException) e).isUserException()) {
-          LOGGER.warn(message + e.getMessage());
+          LOGGER.warn(message, e);
```

Comment:
Why do you change this? I think there is a reason for printing `e.getMessage()` only... @HeimingZ

Reply:
I'm not sure whether we don't need the stackstace indeed.

Reply:
@jun0315 PTAL

Reply:
In the original version:
https://github.com/apache/iotdb/pull/4316/files#diff-94a8b388744fb4d4422cb90e8fd8650803610de8ea6e734b3924f2d3a7cb86f4L2318 (In this page, we need click the `TSServiceImpl.java` `Load diff` button)
it was printed `e.getMessage()`, which I transplanted directly at that time. 
But now I think maybe need the stackstace is a better way?
LGTM

Reply:
Thanks @jun0315